civihr_default_theme/scss/base/_variables.scss

```diff
@@ -29,6 +29,7 @@ $crm-grizzly:         #dcdddd !default;
 $crm-color-neutral:   #949caa !default; // gray shade
 $crm-grayish-blue:    #f4f7f8 !default;
 
+$crm-dark:               #000000 !default;
```

Comment:
Indentation does not match, also in the design the color is not black its, '4c4c5d'. Please use $gray-darker: